lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -220,18 +220,28 @@ class DomRenderer {
     _styleElement = html.StyleElement();
     html.document.head.append(_styleElement);
     final html.CssStyleSheet sheet = _styleElement.sheet;
-
+    final bool isWebKit = browserEngine == BrowserEngine.webkit;
+    final bool isFirefox = browserEngine == BrowserEngine.firefox;
```

Comment:
optional: these could be convenient reusable getters in the `browser_detection.dart` library.